### PR TITLE
QA-180 Fixed new MicroProfile goals.

### PR DESCRIPTION
**Pull request by MattGill98**

### CODE REVIEW

**Comment by mulderbaba** [comment]
jenkins test

**Comment by Pandrex247** [comment]
Jenkins test please

**Comment by Pandrex247** [comment]
Jenkins test please

File: Jenkinsfile-MicroProfile [hunk 1]
```diff
@@ -49,114 +49,114 @@ pipeline {
...
-                sh "mvn -B -V -ff -e clean install -f MicroProfile-JWT-Auth/payara-jwt-auth-arquillian-extension/pom.xml \
+                sh "mvn -B -V -ff -e clean install -f MicroProfile-JWT-Auth/tck-arquillian-extension/pom.xml \
                 -Dpayara.version=${pom.version}"
                 echo '*#*#*#*#*#*#*#*#*#*#*#*#  Installed MP JWT-Auth TCK Arquillian Extension  *#*#*#*#*#*#*#*#*#*#*#*#*#*#*#'
             
                 echo '*#*#*#*#*#*#*#*#*#*#*#*#  Running JWT-Auth TCK  *#*#*#*#*#*#*#*#*#*#*#*#*#*#*#'
-                sh "mvn -B -V -ff -e clean install -f MicroProfile-JWT-Auth/tck-runner/pom.xml \
-                -Dsurefire.rerunFailingTestsCount=2 -Ppayara-micro-managed,full,payara5 -Dpayara.version=${pom.version}"
+                sh "mvn -B -V -ff -e clean verify -f MicroProfile-JWT-Auth/tck-runner/pom.xml \
+                -Dfailsafe.rerunFailingTestsCount=2 -Punstable -Dpayara.version=${pom.version}"
```
Comment (Pandrex247) [inline Jenkinsfile-MicroProfile]: Why unstable?

Comment (MattGill98) [reply Jenkinsfile-MicroProfile]: The profiles were renamed from 'full' and 'base' to 'stable' and 'unstable' to be inkeeping with one of the other TCKs

**Comment by MattGill98** [comment]
jenkins test please

**Pull request merged**